batchlib/analysis/summary.py

```diff
@@ -65,10 +65,11 @@ def write_summary_table(self):
 
         # if this table already exists, then extend the column_dict and the column_names
         if os.path.exists(self.table_path):
-            old_column_dict, old_column_names = read_table(self.table_path)
-            column_names = old_column_names + column_names
-            column_dict = {site_name: old_column_dict[site_name] + column_dict[site_name]
-                           for site_name in column_dict.keys()}
+            assert False  # FIXME this is currently not working, the table is not loaded in the same format..
+            # old_column_dict, old_column_names = read_table(self.table_path)
+            # column_names = old_column_names + column_names
+            # column_dict = {site_name: old_column_dict[site_name] + column_dict[site_name]
+            #                for site_name in column_dict.keys()}
```

Comment:
@constantinpape could you have a look at this? This is currently not working, the parsed columns are not a list but an Index object (from pandas I think). I am also not even sure if we actually ever want to append to an existing table. What would be a scenario when we would use this?

Reply:
The idea was to use it in case we run the cell and pixel analysis.
In any case, I will update all of this for the new table format, so we don't need to bother now.